backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportHeaderGenerator.kt

```diff
@@ -1237,6 +1237,7 @@ private fun Deprecation.toDeprecationAttribute(): String {
     // TODO: consider avoiding code generation for unavailable.
 
     val message = this.message.orEmpty()
+            .replace("\n", "\\n")
```

Comment:
OK as a temporary solution. Any other escaped symbol still fails.
TODO: Consider another approach: getting message "as is"  is dangerous and brute force consistent escaping is complicate.

Reply:
> OK as a temporary solution.

Exactly! That's why I've added a second commit that handles a couple of other symbols. Have you reviewed it?

> TODO: Consider another approach: getting message "as is" is dangerous and brute force consistent escaping is complicate.

Could you clarify?